package.json

```diff
@@ -2,7 +2,7 @@
   "name": "evatr",
   "description": "Validate foreign VAT numbers via German “Bundeszentralamt für Steuern”",
   "author": "Philipp Katz <philipp@philippkatz.de>",
-  "version": "3.2.0",
+  "version": "3.2.1",
```

Comment:
No need to manually change this, it'll happen via NPM automatically. I'll merge and push to NPM once above comment is resolved. Thanks!

Reply:
Thanks for the info; I have reverted this to the previous version.